src/Validation/Validator/RequestTaggedFailed.php

```diff
@@ -38,7 +47,7 @@ public function getId() : string
 
     public function getName() : string
     {
-        return 'Request Tagged as Failed';
```

Comment:
We need to find a better name for the validator. The log output is intended to read like "<validator name> failed with:"

Maybe simply return "Request"?